tests/e2e/steps_downtime.go

```diff
@@ -364,27 +358,24 @@ func stepsThrottledDowntime(consumerName string) []Step {
 					ValPowers: &map[validatorID]uint{
 						validatorID("alice"): 511,
 						validatorID("bob"):   0,
-						validatorID("carol"): 500, // not slashed due to throttling
-					},
-					GlobalSlashQueueSize: uintPointer(1), // carol's slash request is throttled
-					ConsumerChainQueueSizes: &map[chainID]uint{
-						chainID(consumerName): uint(1),
+						validatorID("carol"): 500, // slash packet for carol recv by provider, carol not slashed due to throttling
 					},
 				},
 				chainID(consumerName): ChainState{
 					ValPowers: &map[validatorID]uint{
 						validatorID("alice"): 511,
-						validatorID("bob"):   0,
+						validatorID("bob"):   0, // VSC packet applying bob jailing is also relayed and recv by consumer
 						validatorID("carol"): 500,
 					},
 				},
 			},
 		},
+		// TODO(Shawn): Improve this test to have the consumer retry it's downtime slash, and to assert queue size on consumer.
```

Comment:
Note this TODO, the e2e test ends without retrying the slash packet. This will be addressed in a separate PR

Reply:
Just to confirm, we are removing the Queue size from the chain state, despite this TODO, right? It reads to me as if that would reintroduce it, but maybe I'm missing context

Reply:
Good question, we are removing the queues _from the provider_ in this PR. The queues exist on the consumers as of https://github.com/cosmos/interchain-security/pull/1024, so incoming e2e test will assert queue size on consumer